ledger/ledger.go

```diff
@@ -1128,3 +1128,24 @@ func (l *Ledger) Truncate(utxovmLastID []byte) error {
 	l.xlog.Info("truncate blockid succeed")
 	return nil
 }
+
+// StartPowBlockState 更新powBlockState状态
+func (l *Ledger) StartPowBlockState() {
+	l.powMutex.Lock()
+	defer l.powMutex.Unlock()
+	l.powBlockState = true
+}
+
+// AbortPowBlockState 中断挖矿流程
+func (l *Ledger) AbortPowBlockState() {
```

Comment:
maybe, AbortPowMinning is better for understanding.

Reply:
Yes, all suggestions above are correct and I have updated them. Thank you.